element_animal/export/nwb.py

```diff
@@ -5,22 +5,23 @@
 from element_animal import subject
 
 
-def subject_to_nwb(subject_key):
-    subject_query = subject.Subject & subject_key
+def subject_to_nwb(session_key):
+    subject_query = subject.Subject & session_key
     subject_query = subject_query.join(subject.Subject.Line, left=True)
     subject_query = subject_query.join(subject.Subject.Strain, left=True)
     subject_query = subject_query.join(subject.Subject.Source, left=True)
     subject_info = subject_query.fetch1()
 
     return pynwb.file.Subject(
-        subject_id=subject_info['subject'],
-        sex=subject_info['sex'],
-        date_of_birth=datetime.combine(subject_info['subject_birth_date'],
-                                       datetime.strptime('00:00:00', '%H:%M:%S'
-                                                         ).time()),
+        subject_id=subject_info["subject"],
+        sex=subject_info["sex"],
+        date_of_birth=datetime.combine(
+            subject_info["subject_birth_date"],
+            datetime.strptime("00:00:00", "%H:%M:%S").time(),
+        ),
         description=json.dumps(subject_info, default=str),
-        genotype=' x '.join((subject.Line.Allele * subject.Subject.Line
-                             & subject_key
-                             ).fetch('allele')),
-        species=subject_info['species']
+        genotype=" x ".join(
+            (subject.Line.Allele * subject.Subject.Line & subject_query).fetch("allele")
+        ),
+        #species=subject_info["species"],
```

Comment:
Why not include species here? We could add a conditional to only include if not null. Or are you more concerned with the lack of enforcement around latin naming, preferring to exclude if we can't verify?